frontend/src/features/admin-form/create/builder-and-design/constants.ts

```diff
@@ -95,3 +89,10 @@ export const CREATE_MYINFO_PARTICULARS_DROP_ID = 'create-myinfo-particulars'
 export const CREATE_MYINFO_MARRIAGE_DROP_ID = 'create-myinfo-marriage'
 export const FIELD_LIST_DROP_ID = 'formFieldList'
 export const PENDING_CREATE_FIELD_ID = 'FIELD-PENDING-CREATION'
+
+export const MOCK_MYINFO_IMPLEMENTED_TYPES: MyInfoImplementedTypes[] = [
```

Comment:
Mocks should not be located in app code, should we move this?

Reply:
thanks for the catch! i think my brain short-circuited and i forgot that `mocks/msw` existed even though this is only used there oof